manifests/kubeless.jsonnet

```diff
@@ -2,7 +2,7 @@ local kube = import "kube.libsonnet";
 
 local kubeless = import "vendor/kubeless/kubeless.jsonnet";
 
-local controllerContainer = kubeless.controller.spec.template.spec.containers[0] + { image: "bitnami/kubeless-controller@sha256:939d64b5a50c36036738530d29af678752236e412d06c5eda1831be1a5b588e4" };
+local controllerContainer = kubeless.controller.spec.template.spec.containers[0] + { image: "bitnami/kubeless-controller-manager:" + std.extVar("KUBELESS_VERSION") };
```

Comment:
there is an additional image pointing to `:latest` that we should update in the configMap:

```
local kubeless = import "vendor/kubeless/kubeless.jsonnet";
local k = import "ksonnet.beta.1/k.libsonnet";

local controllerContainer = kubeless.controller.spec.template.spec.containers[0] + { image: "bitnami/kubeless-controller-manager:" + std.extVar("KUBELESS_VERSION") };
local config = kubeless.cfg + k.core.v1.configMap.data({"builder-image": "kubeless/function-image-builder:" + std.extVar("KUBELESS_VERSION")});

kubeless {
  ns: kube.Namespace("kubeless"),
  controller+: {spec+: {template+: {spec+: { containers: [controllerContainer] } } } },
  cfg: config,
}
```

Reply:
thanks, updated!